composer/datasets/utils.py

```diff
@@ -162,3 +163,45 @@ def add_vision_dataset_transform(dataset: VisionDataset, transform: Callable, is
         else:
             dataset.transform = transforms.Compose([dataset.transform, transform])
             log.warning(transform_added_logstring)
+
+
+def create_ffcv_dataset(dataset: IterableDataset,
```

Comment:
Does it need to be an IterableDataset?

```suggestion
def create_ffcv_dataset(dataset: Dataset,
```